packages/playground/src/utils/strings.ts

```diff
@@ -9,11 +9,21 @@ export interface GenerateNameOptions {
   suffix?: string;
 }
 export function generateName(length = 12, options?: GenerateNameOptions): string {
-  const chars = "abcdefghijklmnopqrstuvwxyz";
+  const vowels = "aeiou";
+  const consonants = "bcdfghjklmnpqrstvwxyz";
   const nums = "0123456789";
 
+  const prefix = options?.prefix ?? "";
+  const suffix = options?.suffix ?? "";
+  const remainingLength = length - prefix.length - suffix.length;
+
   return (
-    (options?.prefix ?? "") + randomChoice(chars) + generateString(chars + nums, length - 1) + (options?.suffix ?? "")
+    (options?.prefix ?? "") +
+    randomChoice(consonants) +
+    generateString(vowels + consonants, Math.ceil((remainingLength - 2) / 2)) +
+    generateString(consonants, Math.floor(remainingLength / 3)) +
+    generateString(nums, Math.floor(remainingLength / 6)) +
+    (options?.suffix ?? "")
```

Comment:
I think the changes done here are not needed as we only need to adjust the length where we call the function.